bundle/build

```diff
@@ -59,8 +69,16 @@ for FILE in "${FILES_ETC[@]}"; do
     fi
     cp "$FILE" $TMPDIR/etc
 done
+for FILE in "${FILES_CONTRIB[@]}"; do
+    if [[ ! -f "$FILE" ]]; then
+        echo "File '$FILE' does not exist"
+        exit 1
+    fi
+    cp "$FILE" $TMPDIR/contrib
+done
```

Comment:
On the fence.  Would it be better to go through all the files, noting all that can't be copied, and then exit if one or more can't be?  That way the admin could know all of the issues, rather than fix one, rerun, fix next, rerun, fix next, ...

Your call.

Reply:
I agree and will change it as suggested.